benchmarks/microbenchmarks/README.md

```diff
@@ -153,3 +154,124 @@ To run the test suite:
 ```bash
 python -m unittest discover benchmarks/microbenchmarks/test
 ```
+
+# Training Microbenchmarks
+
+This directory contains tools for benchmarking training performance with low precision datatypes like float8.
+
+## Overview
+
+The training microbenchmarking framework allows you to:
+
+1. Benchmark forward and backward pass performance of models with different precision types
+2. Compare float8 training performance against baseline implementations
+3. Configure various float8 training parameters (scaling types, granularity, etc.)
+4. Measure speedups and performance characteristics
+5. Generate detailed profiling information
+
+## Usage
+
+### Running Training Benchmarks
+
+To run training benchmarks, use the benchmark_runner.py script with a training configuration file:
+
+```bash
+python -m benchmarks.microbenchmarks.benchmark_runner --config benchmarks/microbenchmarks/test/training_benchmark_config.yml
+```
+
+### Configuration File Format
+
+The training benchmark configuration file uses YAML format. Here's an example:
+
+```yaml
+# Configuration for training benchmarks with float8 and other low precision dtypes
+benchmark_mode: "training"
+output_dir: "benchmarks/microbenchmarks/results/training"
+
+# Float8 training specific configurations
+quantization_config_recipe_names:
+  - "float8dq-tensor"
+  - "float8dq-row"
+  - "baseline"  # Always include baseline for comparison
+
+# Float8 training specific configurations
+
+# Training specific configurations
+scaling_type_input: "dynamic"
+scaling_type_weight: "dynamic"
+scaling_type_grad_output: "dynamic"
+scaling_granularity: "tensorwise"
+use_fast_accum: true
+repeat_n: 100  # Number of iterations for benchmarking
+
+model_params:
+  - name: "float8_linear_training"
+    matrix_shapes:
+      - name: "custom"
+        shapes: [
+          [1024, 1024, 1024],  # [m, k, n]
+          [2048, 4096, 1024],
+          [4096, 4096, 1024]
+        ]
+    high_precision_dtype: "torch.bfloat16"
+    use_torch_compile: true
+    torch_compile_mode: "max-autotune"
```

Comment:
I don't think many people are using `max-autotune` for training, I'd recommend changing this to the default (without autotune)